nav2_map_server/launch/map_saver_server.launch.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
```

Comment:
We should document that this exists somewhere... let me think about this. Maybe a docs page on the website or readme or something

Reply:
Updated README, starting from https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-199edd530f2d42ce4b20fe5335734143R11